evm-precompiles/erc20/src/lib.rs

```diff
@@ -274,22 +276,25 @@ where
 		let amount: Balance = amount.saturated_into();
 		let origin: Runtime::AccountId = handle.context().caller.into();
 
-		handle.record_cost(Runtime::GasWeightMapping::weight_to_gas(
-			<Runtime as pallet_assets::Config>::WeightInfo::transfer(),
-		))?;
-		let _ = <pallet_assets_ext::Pallet<Runtime> as Transfer<Runtime::AccountId>>::transfer(
-			asset_id,
-			&origin,
-			&to.clone().into(),
-			amount,
-			false,
-		)
-		.map_err(|e| revert(alloc::format!("ERC20: Dispatched call failed with error: {:?}", e)))?;
+		let destination = to.clone().into();
```

Comment:
not sure if we need rhe `.into()` here - as we dont explicitly define the converted type
```suggestion
		let destination = to.clone();
```

Reply:
this doesn't work without `.into()`
```
--> evm-precompiles/erc20/src/lib.rs:283:18
    |
283 |                 destination: to.clone(),
    |                              ^^^^^^^^^^ expected associated type, found `H160`
    |
    = note: expected associated type `<Runtime as frame_system::Config>::AccountId`
                        found struct `H160`
    = help: consider constraining the associated type `<Runtime as frame_system::Config>::AccountId` to `H160` or calling a method that returns `<Runtime as frame_system::Config>::AccountId`
    = note: for more information, visit https://doc.rust-lang.org/book/ch19-03-advanced-traits.html
help: call `Into::into` on this expression to convert `H160` into `<Runtime as frame_system::Config>::AccountId`
    |
283 |                 destination: to.clone().into(),
    |                                        +++++++
```

Reply:
Ah interesting, ok ignore the comment then 👍